modules/users/manifests/petergoddard.pp

```diff
@@ -1,9 +0,0 @@
-# Creates the petergoddard user
```

Comment:
By deleting this file he'll also need to be removed from staging and production in govuk-secrets, or the next puppet deploy will cause issues

Reply:
Aha, thanks! I'll hold off on merging this until that's done then.

Reply:
Confirmed removed from govuk-secrets: https://github.com/alphagov/govuk-secrets/pull/755